### ISSUE TITLE
Improve message for invalid hiccup tag

### ISSUE THREAD

**Issue by milelo** [milelo/electric-hiccup#11]
Add the tag to the message

**Comment by milelo** [comment]
Addressed in: 616f6479a452d2f6dd013e8e2aaf28d7b21fc269

**Comment by milelo** [comment]
Released in:
```clojure
        milelo/electric-hiccup {:git/url "https://github.com/milelo/electric-hiccup"
                                :git/tag "v0.4.0-alpha"
                                :git/sha "f29103a"}
```
